internal/config/5_real_config.go

```diff
@@ -13,7 +13,7 @@ import (
 func init() {
 	if clowder.IsClowderEnabled() {
 		cfg := clowder.LoadedConfig
-
+		viper.Set("sources.appId", 11)
```

Comment:
this doesn't work (my bad) sorry
```suggestion
		viper.Set("restEndpoints.sources.appId", 11)
```

Reply:
Thank you!